rxbinding-support-v4/src/androidTest/java/com/jakewharton/rxbinding2/support/v4/widget/RxDrawerLayoutTest.java

```diff
@@ -79,18 +80,26 @@
   }
 
   @Test public void open() {
-    final Action1<? super Boolean> open = RxDrawerLayout.open(view, RIGHT);
+    final Consumer<? super Boolean> open = RxDrawerLayout.open(view, RIGHT);
 
     instrumentation.runOnMainSync(new Runnable() {
       @Override public void run() {
-        open.call(true);
+        try {
+          open.accept(true);
+        } catch (Exception e) {
+          fail();
```

Comment:
In ae53ed20efc862487084622a3ad643bc83b5bb6d I had similar issue with checked exception.
I've introduced `UnsafeRunnable` there. I wonder which approach is cleaner and which should we use.
@JakeWharton ?